### DISCUSSION TITLE
When OPA v0.46.0 should be released approximately?

### DISCUSSION THREAD

**Discussion by RonAmihai** [open-policy-agent/community#291] [OPA and Rego] [answered]
It seems like the planned v0.46.0 fixes some vulnerabilities of v0.45.0. 

Do you know when v0.46.0 is scheduled to be released?

**Comment by anderseknert** [comment]
Hi Ron! OPA follows a monthly release cadence +/- a few days.. So my guess would be v0.46.0 is released sometime next week.

**Comment by srenatus** [comment, marked as answer]
First week of November.

But I'm unaware of any critical vulnerabilities. We'd do a patch release earlier for those. If you've got something in mind there please reach out to the security mailing list.
